Source/Orts.Formats.Msts/ActivityFile.cs

```diff
@@ -1133,6 +1157,60 @@ public ORTSWeatherChange(STFReader stf)
         }
     }
 
+    public class ActivitySound
+    {
+        public string ORTSActSoundFile;
+        public int ORTSActSoundFileType;
+        public int TileX;
+        public int TileZ;
+        public float X;
+        public float Y;
+        public float Z;
+        public ActivitySound(STFReader stf, string fileName)
+        {
+            stf.MustMatch("(");
+            stf.ParseBlock(new STFReader.TokenProcessor[] {
+                new STFReader.TokenProcessor("ortsactsoundfile", ()=>
+                {
+                    stf.MustMatch("(");
+                    var tempString = stf.ReadString();
+                    ORTSActSoundFile =Path.Combine(Path.Combine(Path.GetDirectoryName(Path.GetDirectoryName(fileName)), "SOUND"), tempString);
+                    var ORTSActSoundFileTypeString = stf.ReadString();
+                    switch (ORTSActSoundFileTypeString)
+                        {
+                            case "Everywhere":
+                                ORTSActSoundFileType = 0;
+                                break;
+                            case "Cab":
+                                ORTSActSoundFileType = 1;
+                                break;
+                            case "Pass":
+                                ORTSActSoundFileType = 2;
+                                break;
+                            case "Ground":
+                                ORTSActSoundFileType = 3;
+                                break;
+                            case "Location":
+                                ORTSActSoundFileType = 4;
+                                break;
+                            default:
+                                ORTSActSoundFileType = 0;
+                                break;
+                        }
```

Comment:
This looks like it should be an enum, as then you can skip the whole switch and just parse straight into the value.

Reply:
Yeah, that looks like the kind of thing I mean regarding enum.

If you make the change locally and push it on the same branch as this PR (strongly recommended) the PR will update automatically, including emailing me about it.